src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.WebKey.Tests/WebKeyOperationsTest.cs

```diff
@@ -21,6 +22,25 @@ public void ClearMemoryTest()
             }
         }
 
+        [Fact]
+        public void JwkEqualsTests()
```

Comment:
This is covered in the first case that fields are empty, except for KeyOps. Other fields are tested for that.